src/assets/script.js

```diff
@@ -377,6 +377,7 @@ function highlightCodeBlocks() {
             const language = codeBlock.result.language
             const displayLanguage = displayLanguageList[language] || language
             const languageWrapper = document.createElement('p')
+            
             languageWrapper.classList.add('bg-primary', 'd-inline-block', 'p-2', 'mb-0', 'font-weight-bold')
```

Comment:
You need to update your branche after upgrading to the newest version of Boosted:

> All our helpers and utilities have also been adapted to the new color modes mechanism and automatically switch their colors and background colors depending on the current color mode. As a result, you should for instance now rather use .text-bg-{color} classes instead of .bg-{color} classes when possible for automatic contrast.
(see: https://boosted.orange.com/docs/5.3/migration/)